components/dashboard/src/start/StartPage.tsx

```diff
@@ -105,6 +107,17 @@ export function StartPage(props: StartPageProps) {
                 )}
                 {error && <StartError error={error} />}
                 {props.children}
+                {props.showLatestIdeWarning && (
+                    <InfoBox className="mt-4">
```

Comment:
**suggestion**: @mustard-mh Following our discussion about the alwert component in dak layout, I've updated the component colors in the [current exploration design file](https://www.figma.com/file/y3KbNYhawcNvN8DUYpePZQ/Improve-warning-alert-component-design-%237613?node-id=807%3A13). Posting below two exports to get a sense of how this could looks like.

| Alert componen in light theme | Alert componen in dark theme |
|-|-|
| <img width="454" alt="Frame 376" src="https://user-images.githubusercontent.com/120486/158857456-db2399b1-fbf6-4bad-9b33-dd450e51bba6.png"> | <img width="454" alt="Frame 377" src="https://user-images.githubusercontent.com/120486/158857463-4042b408-e0dc-4906-b662-51c64daccf3f.png"> |

**issue**: The issue with using links within the alert component always get tricky when you switch to dark layout and if you want to use the blue color we're using for links.

Reply:
In summary, `dark:bg-yellow-700` for the background and `dark:bg-yellow-60` for the text color. For the icon I've been using `yellow-900` for the fill color but you can also get these variables from the design specs, too.

Thanks for the ping @mustard-mh! 🏓